apps/api/src/server/repositories/representation.repository.js

```diff
@@ -582,6 +609,28 @@ export const getApplicationRepresentationForDownload = async (caseId, skip, batc
 	});
 };
 
+/**
+ * @param {number} caseId
```

Comment:
Might be good to have a quick explanation of what "publishable" means in this context - can be inferred from the code, but a comment might be helpful.